lineman/app/models/base_endpoints_interface.coffee

```diff
@@ -0,0 +1,22 @@
+angular.module('loomioApp').factory 'BaseEndpointsInterface', (RestfulClient) ->
```

Comment:
I think it makes Records and the related classes potentially way more complex if there is a similar but different thing in there. It's about as complex as I can handle already.

Could this just be InvitationsClient or InvitaitonsEndpoint?  If it's going to be in Records then I think it needs to be actually using the record store, otherwise it will get in the way of the continuing development of the Records concept.

Can you remind me why this isn't a normal kind of record like everything else?